tests/devapps/NetCoreTestApp/Program.cs

```diff
@@ -275,6 +279,25 @@ 1. IWA
                             }
 
                             break;
+
+                        case 12: // acquire token interactive with WamBroker
+                            {
+                                var optionsbroker = new SystemWebViewOptions()
```

Comment:
Why are these added? They don't affect the broker?

Reply:
Case 12 is for using the old/legacy WamBroker